gui/default/syncthing/core/syncthingController.js

```diff
@@ -561,6 +594,13 @@ angular.module('syncthing.core')
                         data[id].outbps = 0;
                     }
                 }
+                // Clean out stale addresses from probing results
+                for (var address in $scope.remoteGUICache) {
+                    console.log("clean test", address);
```

Comment:
Debugging leftover, should be removed.

Reply:
Still left in latest commit, please remove the `console.log("clean test", address);` line.